recipes/rlr/LICENSE

```diff
@@ -0,0 +1,19 @@
+The MIT License (MIT)
```

Comment:
Where did this license come from? It's not the one in the [upstream repo](https://github.com/dedupeio/rlr/blob/master/COPYING) and it's not an Apache 2.0 license, which what the `setup.py` for this version says it is. I'd be ok with having a generic Apache 2.0 license file here.